tests/framework/microvm.py

```diff
@@ -151,7 +152,7 @@ def __init__(
         self._screen_pid = None
         self._screen_log = None
 
-        self.time_api_requests = True
+        self.time_api_requests = global_props.host_linux_version != "6.1"
```

Comment:
do we need a TODO/issue here?

Reply:
I was going to just leave a comment on the internal ticket about investigating the 6.1 boot time/snapshot regressions.